app/src/main/java/comp3350/mealbuddy/persistence/DataAccessObject.java

```diff
@@ -780,8 +780,12 @@ public String addDay(String userName, int dayOfYear) {
     public Day getDay(String userName, int dayOfYear) {
         Day day = new Day(dayOfYear);
         try {
-            day.exercises = getDayExercises(userName, dayOfYear);
```

Comment:
I don't see the purpose of making a deep copy here.  getDayExercises already is creating a new list from the db

Reply:
Oh i see